contributors/design-proposals/api-proposal-design-template.md

```diff
@@ -0,0 +1,143 @@
+# Description of your API
+
+Initial API version: v1alpha1
+
+## Owners
+
+SIG owner: list of SIG that will own the design, implementation, and support
+Individual owners: list of individual GitHub users that will own the design, implementation, and support
+
+## Abstract
+
+4-8 sentence description of the problem and solution.  Bullet points welcome.
+
+## Motivation
+
+4-8 sentence of the current state and why it is insufficient.
+
+## Use cases
+
+Bullet point list of use cases this proposal is meant to address.
+
+- As a cluster administrator...
+- As a application developer...
+- As a Kubernetes extension developer...
+
+## Requirements
+
+Bullet point list of new API requirements.  Requirements will in part be driven by use
+cases, but may have additional engineering or interoperability requirements:
+
+- User requirement
+- Engineering requirement
+- Interoperability requirement
+
+## Overview subsequent versions
+
+Subsequent versions will require their own API proposals, however it is important to include
+the plans for them here to provide a broader picture.
+
+### Beta
+
+*Use cases*
+
+- Use cases for transitioning to Beta
+
+*Requirements*
+
+- Additional requirements for transitioning to Beta
+
+### GA
+
+*Use cases*
+
+- Use cases for transitioning to Beta
```

Comment:
s/Beta/GA/

Do we really need beta & GA requirements up front? If people knew what they needed to do they would just do it to begin with.